### DISCUSSION TITLE
Remove Contacts section

### DISCUSSION THREAD

**Discussion by EquivalentFlow** [morethanmin/morethan-log#135] [Q&A]
Hi, how can I remove the Contact or Service sections?

**Comment by elow21** [comment]
Hey @EquivalentFlow 

I just tried this method. You can remove the Service section completely:

- Remove line 13 to 18 in `site.config.js`
- Remove line 10 in `/src/routes/Feed/index.tsx`
- Removing this file `src/routes/Feed/ServiceCard.tsx`
